easybuild/easyconfigs/w/WRF/WRF-4.4-foss-2022a-dmpar.eb

```
@@ -0,0 +1,52 @@
+name = 'WRF'
+version = '4.4'
+buildtype = "dmpar"
+versionsuffix = '-%s' % buildtype
+
+homepage = 'https://www.wrf-model.org'
+description = """The Weather Research and Forecasting (WRF) Model is a next-generation mesoscale
+ numerical weather prediction system designed to serve both operational forecasting and atmospheric
+ research needs."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'opt': False}  # don't use agressive optimization, stick to -O2
+
+source_urls = ['https://github.com/wrf-model/WRF/archive/']
+# sources = ['v%(version)s.tar.gz']
+sources = [{
+    'filename': 'v4.4.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/wrf-model',
+        'repo_name': 'WRF',
+        'tag': 'v4.4',
+        'clone_into': 'WRF-4.4',
+        'recursive': True,
+    },
+}]
+patches = [
+    'WRF-%(version)s_netCDF-Fortran_separate_path.patch',
+]
+# The problem: sources from git clone have different checksum every time
+# checksums = [
+#     '0d68df19766ad03041126c1531cb115e2d2a69043df96314fd30d897cfca9e4c',  # v4.4.tar.gz
+#     '0e37c8a7bb4d25947083bdb6d0f2a9f4fdb825c88f6cb10c59b7580fe3d129ff',  # WRF-4.4_netCDF-Fortran_separate_path.patch
+# ]
```

To make the style checker happy, I would use `None` for the checksum from the `git clone` command.
```suggestion
checksums = [
    None,  # v4.4.tar.gz
    '0e37c8a7bb4d25947083bdb6d0f2a9f4fdb825c88f6cb10c59b7580fe3d129ff',  # WRF-4.4_netCDF-Fortran_separate_path.patch
]
```